bodo/tests/test_df_lib/test_direct_series_methods.py

```diff
@@ -0,0 +1,99 @@
+import pandas as pd
+import pytest
+
+import bodo.pandas as bd
+from bodo.tests.utils import _test_equal
+
+
+def gen_direct_series_test(name, arg_sets):
+    """
+    Generates a parameterized test case for direct Series.<name> method.
+    """
+
+    @pytest.mark.parametrize(
+        "args, kwargs",
+        arg_sets,
+        ids=[
+            f"{name}-args{args}-kwargs{sorted(kwargs.items())}"
+            for args, kwargs in arg_sets
+        ],
+    )
+    def test_func(args, kwargs):
+        df = pd.DataFrame(
+            {
+                "A": pd.array([1.445, 2.12, None, -4.133], dtype="Float64"),
+                "B": pd.array([1, 2, 3, 4], dtype="Int64"),
+            }
+        )
+        bdf = bd.from_pandas(df)
+
+        keys = ["A", "B"]
+
+        for key in keys:
```

Comment:
Looks like there is some duplicated code in test generation that could be refactored in a follow up PR.